include/dl/dl_util.h

```diff
@@ -52,13 +53,15 @@ typedef enum
 		filetype     - Type of file to read, see dl_util_file_type_t.
 		out_instance - Pointer to fill with read instance.
 		out_type     - TypeID of instance found in file, can be set to 0x0.
+		allocator 	 - Allocator for doing temp file allocations
 
 	Returns:
 		DL_ERROR_OK on success.
 */
 dl_error_t dl_util_load_from_file( dl_ctx_t    dl_ctx,       dl_typeid_t         type,
 								   const char* filename,     dl_util_file_type_t filetype,
-								   void**      out_instance, dl_typeid_t*        out_type );
+								   void**      out_instance, dl_typeid_t*        out_type,
+								   dl_allocator *allocator = 0x0 );
```

Comment:
I would like to keep default-values out of the interface as this breaks c-compat.

Reply:
This also need doc that says that 0 is valid an what that means.

Reply:
Ait! I will remove default-value and say that 0 is valid!